### PR TITLE
fix: added shard card experiment tracking to funded borrower profile component

### PR DESCRIPTION
**Pull request by edwinf-barreram**
# Ticket 

[MARS-198](https://kiva.atlassian.net/browse/MARS-198)

### CODE REVIEW

File: src/pages/BorrowerProfile/BorrowerProfile.vue [hunk 1]
```diff
@@ -515,7 +515,7 @@ export default {
 		}
 
 		// EXP-MARS-143-Jul2022
-		this.shareCardLanguageVersion = this.$route.query?.scle;
+		this.shareCardLanguageVersion = this.$route.query?.utm_campaign;
```
Comment (mcstover) [inline src/pages/BorrowerProfile/BorrowerProfile.vue:518]: I think this should be gated with a check for `scle` within the utm_campaign param

File: src/components/Checkout/SocialShareV2.vue [hunk 2]
```diff
@@ -129,16 +129,17 @@ export default {
 		shareLink() {
 			const base = `https://${this.$appConfig.host}`;
 			if (this.selectedLoan.id) {
-				return `${base}/invitedby/${this.lender.inviterName}/for/${this.selectedLoan.id}?utm_content=${this.utmContent}&scle=${this.shareCardLanguageVersion}`; // eslint-disable-line max-len
+				return `${base}/invitedby/${this.lender.inviterName}/for/${this.selectedLoan.id}?utm_content=${this.utmContent}`; // eslint-disable-line max-len
 			}
-			return `${base}?utm_content=${this.utmContent}&scle=${this.shareCardLanguageVersion}`;
+
+			return `${base}?utm_content=${this.utmContent}&utm_campaign=social_share_checkout_variant_scle_${this.shareCardLanguageVersion}`; // eslint-disable-line max-len
```
Comment (mcstover) [inline src/components/Checkout/SocialShareV2.vue:135]: The utm_campaign values on this page should not contain `variant`.

File: src/pages/BorrowerProfile/FundedBorrowerProfile.vue [hunk 3]
```diff
@@ -326,6 +378,14 @@ export default {
 		if (enabled && exp.version === 'b') {
 			this.enabledExperiment = true;
 		}
+
+		// EXP-MARS-143-Jul2022
+		this.shareCardLanguageVersion = this.$route.query?.utm_campaign;
```
Comment (mcstover) [inline src/pages/BorrowerProfile/FundedBorrowerProfile.vue:383]: this should also be gated on the presence of the string "scle" in the utm_campaign value

**Review by mcstover — APPROVED** [review]
Please verify in DEV

**Review by christian14b — APPROVED** [review]

File: src/pages/BorrowerProfile/BorrowerProfile.vue [hunk 4]
```diff
@@ -514,13 +514,15 @@ export default {
...
 		}
 
-		// EXP-MARS-143-Jul2022
-		this.shareCardLanguageVersion = this.$route.query?.scle;
-		this.$kvTrackEvent(
-			'Thanks',
-			'EXP-MARS-143-Jul2022',
-			this.shareCardLanguageVersion
-		);
+		if (this.$route.query?.utm_campaign.includes('scle')) {
```
Comment (dyersituations) [inline src/pages/BorrowerProfile/BorrowerProfile.vue:517]: If `utm_campaign` can be undefined, you probably want optional chaining before `includes` as well.

File: src/pages/BorrowerProfile/FundedBorrowerProfile.vue [hunk 5]
```diff
@@ -326,6 +378,16 @@ export default {
 		if (enabled && exp.version === 'b') {
 			this.enabledExperiment = true;
 		}
+
+		if (this.$route.query?.utm_campaign.includes('scle')) {
```
Comment (dyersituations) [inline src/pages/BorrowerProfile/FundedBorrowerProfile.vue:382]: Same.

**Review by dyersituations — APPROVED** [review]

File: src/pages/BorrowerProfile/FundedBorrowerProfile.vue [hunk 6]
```diff
@@ -231,7 +232,25 @@ export default {
 					name: 'description',
 					content: `A loan helped ${this.loan.use}`,
 				}
-			]
+			].concat(
```
Comment (eddieferrer) [inline src/pages/BorrowerProfile/FundedBorrowerProfile.vue:235]: This is fine as a minimum but I think the funded profile should have the same meta fields as the borrower profile. For example 'og:type', 'og:image', 'twitter:image', 'twitter:title'

**Review by eddieferrer — APPROVED** [review]

**Review by mcstover — APPROVED** [review]

**Pull request merged**